tests/integration/suite/test_system_project.py

```diff
@@ -71,3 +73,14 @@ def test_system_project_cant_be_deleted(admin_mc, admin_cc):
         admin_mc.client.delete(system_project)
     assert e.value.error.status == 405
     assert e.value.error.message == 'System Project cannot be deleted'
+
+
+def test_system_namespaces_default_svc_account(admin_mc):
```

Comment:
The test should use the rancher setting `system-namespaces` so we know what rancher considers a system namespace. This will also help with upkeep of the tests as the setting changes.